webapp/templates/jury/user.html.twig

```diff
@@ -116,6 +116,16 @@
                         {% endif %}
                     </td>
                 </tr>
+                <tr>
+                    <th>Last API login</th>
+                    <td>
+                        {% if user.lastApiLogin %}
+                            {{ user.lastApiLogin | printtime('D d M Y H:i:s T') }}
```

Comment:
Can't we use a default (https://twig.symfony.com/doc/2.x/filters/default.html) here?

Reply:
Can we? I don' think we can. This would become:

```
{{ user.lastApiLogin | default(whatgoesherethatmakessensebutwecanstillcallprinttimeon) | printtime('D d M Y H:i:s T') }}
```

If you know a good default tell me, but null isn't one, since that would print the current date/time

Reply:
I had hoped `default('-')` but that might need a change to the `printtime` function to handle that specific case.